improver_tests/acceptance/SHA256SUMS

```diff
@@ -658,14 +664,20 @@ d197c8f5f1a08caddaa5efabac4a52d75185889d64d2c4b72cbba2f2c0f42a20  ./weighted_ble
 30062d9c23527634a20cae6e7cf7a163aaee361b06ee800c8bea70a7a0f412ec  ./weighted_blending/three_models/nc/20190101T0400Z-PT0001H00M-precip_rate.nc
 850d48dfea1d1e5b696324f0e57a52fce3e3128008c58ccaca82e31598196c67  ./weighted_blending/three_models/ukvx/20190101T0400Z-PT0002H00M-precip_rate.nc
 63b4691e4d7e246b58e01c84f3c8cf82a6cdc7e14068ddd96418e8c0054e2bcf  ./weighted_blending/weights_from_dict/kgo.nc
-124a429618807548ee8890a2e1d4275be0e1e7977d9cfc804e55892bb045d323  ./wet-bulb-temperature-integral/basic/input.nc
-e3bef1754689e3ed8760774bfd293f0fee8ee647c76b308cf7a5a75fac22369e  ./wet-bulb-temperature-integral/basic/kgo.nc
+c2b71855f260544ca75aaa5900f3297b58cee1b3b64eec5805662e1927d5fdab  ./wet-bulb-temperature-integral/basic/with_id_attr/input.nc
+651c80cd4ab7c720c59c6c0c2c4a2ade85aee81c2ab0c697b577ca87a36d57f2  ./wet-bulb-temperature-integral/basic/with_id_attr/kgo.nc
+124a429618807548ee8890a2e1d4275be0e1e7977d9cfc804e55892bb045d323  ./wet-bulb-temperature-integral/basic/without_id_attr/input.nc
```

Comment:
Again, I don't think we need two versions of the input

Reply:
Duplication removed.